### ISSUE TITLE
Reloadium experienced a fatal error when running in debug mode (doesn't apply to regular run)

### ISSUE THREAD

**Issue by Joseph-D-Bradshaw** [reloadware/reloadium#133]
## Describe the bug*
Start a Python server that relies on open source project Shyft, make a distributed time series server instance.
Works fine in Reloadium "run" mode, but not in "debug" mode.

## To Reproduce
1. Install [Shyft](https://gitlab.com/shyft-os/shyft) in a virtual env if needed
2. Activate the environment
3. Make a python script with the following code:
    ```python
    from shyft.time_series import DtsServer
    s = DtsServer()
    s.start_server()
    ```
4. Run the snippet with Reloadium in debug mode
5. See the error message "Reloadium experienced a fatal error and has to quit"

## Expected behavior
The script would exit with status code 0 to indicate no issues.

## Desktop or remote (please complete the following information):**
 - OS: Fedora
 - OS version: 37
 - M1 chip: no
 - Reloadium package version: N/A (I only installed plugin through PyCharm marketplace.. not sure about package..)
 - PyCharm plugin version: 1.0.1
 - Editor: PyCharm Professional
 - Python Version: 3.11.2
 - Python Architecture: 64bit
 - Run mode: Debug

## Additional context
I am guessing it could due to Boost python, exposing C++ functionality to Python layer of this library.

**Comment by bearpaddy** [comment]
Got the same issue as well, using pycharm communcity(looks like it's version 1.1.0)

**Comment by dkrystki** [comment]
I've tried reproducing this but I'm getting 
```
ERROR: Could not find a version that satisfies the requirement shyft (from versions: none)
ERROR: No matching distribution found for shyft
```
When trying to install shift for python 3.11
By looking at the package files https://pypi.org/project/shyft/#files I can see that it's only available for 3.10

I've tried running it under 3.10 but I'm getting 
```
from shyft.time_series._time_series import
ImportError: libpython3.10.so.1.0: cannot open shared object file: No such file or directory
```
That is without reloadium

**Comment by dkrystki** [comment]
Fixed in Reloadium 1.2.1 and PyCharm plugin 1.2.1
